src/elements/content-preview/ContentPreview.js

```diff
@@ -74,6 +74,7 @@ type Props = {
     onDownload: Function,
     onLoad: Function,
     onNavigate: Function,
+    onVersionChange: (versionId?: string, isCurrentVersion?: boolean) => void,
```

Comment:
Should the function signature move to its own named flow type, since it's used in several disparate components?

Reply:
sure, will be a part of the final PR.